website/static/css/fangorn.css

```diff
@@ -355,3 +355,9 @@ a.fangorn-toolbar-icon {
     display: inline-block;
     background: url('/static/img/hgrid/fontawesome/file-text-o.png');
 }
+
+/* file pop up model text  */
```

Comment:
Since you need to fix the other typos, might as well fix styling as well: `modal` not `model`